lib/pos_mempool.go

```diff
@@ -129,6 +140,14 @@ type PosMempool struct {
 	// feeEstimator is used to estimate the fee required for a transaction to be included in the next block
 	// based off the current state of the mempool and the most n recent blocks.
 	feeEstimator *PoSFeeEstimator
+
+	// augmentedBlockViewRefreshIntervalMillis is the frequency with which the augmentedLatestBlockView is updated.
+	augmentedBlockViewRefreshIntervalMillis uint64
```

Comment:
I'd like to start this value at 10ms. I don't think it will cause locking issues here the way it does with legacy mempool because here we hold the write-lock only to make a pointer-copy. In theory, that means we really could set this value to zero and probably be fine. But 10ms seems like a good middle-ground that users hopefully won't really notice.